package.json

```diff
@@ -51,7 +51,7 @@
     "polished": "^3.4.1",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-loadable": "^5.5.0",
+    "react-loadable-ssr-addon": "https://github.com/schachte/react-loadable-ssr-addon#a88593a342fda85a79bf0dfc9f800fdff0693719",
```

Comment:
I don't think revelio should bringing this in. I think ace should proxy.

Reply:
We use it directly in Revelio. 
https://github.com/connexta/revelio/pull/135/files#diff-1ac4767f66c2bb3895e13cd33163c951R10